scripts/qa_crowbarsetup.sh

```diff
@@ -2008,7 +2008,9 @@ function reboot_nodes_via_ipmi()
                     ipmitool -H $ip -U root -P $pw power on
                     sleep $((10 + RANDOM % 15))
                 fi
-                ipmitool -H $ip -U root -P $pw power reset) &
+                ipmitool -H $ip -U root -P $pw power off
+                sleep 5
```

Comment:
@kbaikov can you find out if there is a query command from `ipmitool` which we can use to avoid the `sleep`?
it would be better to have that in a busy loop (look for the function `wait_for`)